src/prefect/server/api/ui/task_runs.py

```diff
@@ -187,3 +187,22 @@ async def read_dashboard_task_run_counts(
         buckets[index].failed = row.failed_count
 
     return buckets
+
+
+@router.post("/count")
+async def read_task_run_counts_by_state(
+    flows: schemas.filters.FlowFilter | None = None,
+    flow_runs: schemas.filters.FlowRunFilter | None = None,
+    task_runs: schemas.filters.TaskRunFilter | None = None,
+    deployments: schemas.filters.DeploymentFilter | None = None,
```

Comment:
excited for this to be untrue, but we have to support 3.8+ so we can't use 3.10+ syntax yet 😢

Reply:
I realized what I had done immediately after I opened the PR 😞

Reply:
"but tests are passing on my machine!"